### PR TITLE
AdminConsole view changes

### PR DESCRIPTION
**Pull request by israelCyabukombe**
Modified a bit the site.css

### CODE REVIEW

File: app/wama.web/wwwroot/css/site.css [hunk 1]
```diff
@@ -8,6 +8,9 @@ input#check-in-user-id {
     text-align: center;
 }
 
+.large{
```
Comment (dubeme) [inline app/wama.web/wwwroot/css/site.css]: This will go eventually, since it's not needed.

File: app/wama.web/Views/AdminConsole/Index.cshtml [hunk 2]
```diff
@@ -5,11 +5,65 @@
...
 }
 
-Todays activities
 
-<h1>Patron check-ins: 45</h1>
-<h1>Employee check-ins: 5</h1>
\ No newline at end of file
+<h1 class="panel"> Today's Activities</h1>
+<h2 class="panel" id="utc1"></h2>
+<head>
```
Comment (dubeme) [inline app/wama.web/Views/AdminConsole/Index.cshtml:14]: I don't think we need <head> and <body> tag?

File: app/wama.web/Views/AdminConsole/Index.cshtml [hunk 3]
```diff
@@ -5,11 +5,65 @@
...
+
+    <title>Admin template</title>
```
Comment (dubeme) [inline app/wama.web/Views/AdminConsole/Index.cshtml:16]: Remember to visit the asp.net mvc6 documentation on how the views a built.

File: app/wama.web/Views/AdminConsole/Index.cshtml [hunk 4]
```diff
@@ -5,11 +5,65 @@
...
+</head>
+<body>
+    <div id="container">
+        <div id="box">
+            <div class="panel panel-primary">
+                <h4 class="panel panel-heading">Patron check-ins</h4>
+
+                <div class="panel-footer">
+                    Currently checked-in:
+                    <div class="large">45 </div>
```
Comment (dubeme) [inline app/wama.web/Views/AdminConsole/Index.cshtml:27]: Why introduce a new CSSS class?

Comment (dubeme) [reply app/wama.web/Views/AdminConsole/Index.cshtml:27]: Look into using a Bootstrap CSS class OR another html tag, to achieve the same effect.

File: app/wama.web/Views/AdminConsole/Index.cshtml [hunk 5]
```diff
@@ -5,11 +5,65 @@
...
+                    <div class="panel-footer"> View Details</div>
+                </a>
+            </div>
+        </div>
+        <div id="box">
+            <div class="panel panel-primary">
+                <h4 class="panel panel-heading">Employee Check-ins</h4>
+                <div class="panel-footer">
+                    Currently checked-in: <div class="large">46</div>
+                </div>
```
Comment (dubeme) [inline app/wama.web/Views/AdminConsole/Index.cshtml:39]: Same here

File: app/wama.web/Views/AdminConsole/Index.cshtml [hunk 6]
```diff
@@ -5,11 +5,65 @@
...
+                    <div class="panel-footer"> View Details</div>
+                </a>
+            </div>
+        </div>
+        <div id="box">
+            <div class="panel panel-primary">
+                <h4 class="panel panel-heading">Other Account Info</h4>
+                <div class="panel-footer">
+                    Accounts Pending Approval: <div class="large">47</div>
+                </div>
```
Comment (dubeme) [inline app/wama.web/Views/AdminConsole/Index.cshtml:50]: Same here, also every other place on this file that uses the cs class you added.

File: app/wama.web/wwwroot/css/site.css [hunk 7]
```diff
@@ -78,9 +81,9 @@ td {
 
 /* Sidebar navigation */
 .nav-sidebar {
```
Comment (dubeme) [inline app/wama.web/wwwroot/css/site.css]: Is there any reason for the change here? 
If not, please revert these changes.

**Review by dubeme — CHANGES_REQUESTED** [review]
Looking good, but needs some more work. Please see comment

File: app/wama.web/wwwroot/css/site.css (hunk 1 above)
Comment (israelCyabukombe) [reply app/wama.web/wwwroot/css/site.css]: Okay. Would you like the numbers to be more visible (larger) in some sort or on the same line as the indication. eg 
Employees check in: 35

File: app/wama.web/Views/AdminConsole/Index.cshtml (hunk 3 above)
Comment (israelCyabukombe) [reply app/wama.web/Views/AdminConsole/Index.cshtml:16]: okay

File: app/wama.web/wwwroot/css/site.css (hunk 7 above)
Comment (israelCyabukombe) [reply app/wama.web/wwwroot/css/site.css]: I must have added that earlier because it was commented out and all I did was un-comment  it and it came up as a change. I'll get rid of the entire block.

File: app/wama.web/Views/AdminConsole/Index.cshtml (hunk 4 above)
Comment (israelCyabukombe) [reply app/wama.web/Views/AdminConsole/Index.cshtml:27]: I'll look into that
